process/process_areas/requirements_engineering/guidance/requirements_process_reqs.rst

```diff
@@ -24,15 +24,18 @@ Process Requirements
    :satisfies: wf__req__stkh_req, wf__req__feat_req, wf__req__comp_req
    :complies: std_req__iso26262__support_6431, std_req__iso26262__support_6432
 
-   Requirements shall be hierarchically grouped into different levels.
+   Requirements shall be hierarchically grouped into three levels.
 
    Following levels are defined:
 
       * Stakeholder requirement
       * Feature requirement
       * Component requirement
+
+   Additionally there shall be the following logical groups of requirements:
       * Assumption of use requirement
       * Process requirement
```

Comment:
"gd_req" is a process requirement? What about e.g. "wf__req"? Is "process requirement" an undefined group here?